uwblueprint/website-bp
# Add Firebase auth & hook up app form

### Implementation description

- Created login page for Firebase OAuth (and implemented the login call)
- User session will be stored in localStorage, with manual token refresh on timeout
- Hooked up the read-only app form - some components are a bit buggy but will be fixed in upcoming PRs
- Logout button not yet implemented - also to be done in future PR



### Steps to test

1. Navigate to `http://localhost:3000/admin/student-details/{id}`, it should require user authentication
2. Should see user app form data show up on the read-only form



### What should reviewers focus on?

- No 🐛 s

### Checklist
